server/db/models/LineItem.js

```diff
@@ -0,0 +1,13 @@
+const Sequelize = require('sequelize');
+const { INTEGER } = Sequelize;
+const db = require('../database');
+
+const LineItem = db.define('lineItem', {
+  quantity: {
+    type: INTEGER,
+    allowNull: false,
+    defaultValue: 1
+  }
+});
```

Comment:
A nice to have is to have a virtual column for calculating the price of this line item adjusted for quantity.